Makefile

```
@@ -9,15 +9,23 @@ API_FILTER := --filter=yap-api
 .PHONY: help install clean dev build check check-biome test unit e2e preflight ready-to-push \
 	portfolio-dev portfolio-build portfolio-check \
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)